bincommon/commonflags.go

```diff
@@ -119,7 +119,9 @@ var (
 
 // SharedMain is the common part of main from fortio_main and fcurl.
 func SharedMain(usage func(io.Writer, ...interface{})) {
-	flag.Var(&headersFlags, "H", "Additional `header`(s)")
+	flag.Var(&headersFlags, "H", "Additional http headers or grpc custom metadata (some grpc reserved or "+
+		"illegal keys are not supported, see https://grpc.io/docs/what-is-grpc/core-concepts/#metadata). "+
```

Comment:
What's an example of illegal keys and what do you mean not supported, what happens? 
(in other words; is that warning/info necessary or can we just have the short version, something like

"Additional http header(s) or grpc metadata. Multiple `key:value` pairs can be passed using multiple -H."

run `release/updateFlags.sh` to see the actual help btw, and using \` in the text labels the value

Reply:
I will add some clearer examples and explanation

Reply:
I simplified further / as above